### DISCUSSION TITLE
a/optimize-website-ssllabs-rating

### DISCUSSION THREAD

**Discussion by giscus** [sincerefly/giscus-data#15] [Announcements]
# a/optimize-website-ssllabs-rating

https://blog.yasking.org/a/optimize-website-ssllabs-rating.html

**Comment by sleepm** [comment]
https://segmentfault.com/a/1190000011097942
dns caa DNS证书颁发机构授权
不过在测试结果上也就是多个 yes

**Reply by sincerefly** [reply]
👍  学到了，文章中推荐的 [https://sslmate.com/caa/](https://sslmate.com/caa/) 很好用
